### PR TITLE
Add napari-zelda recipe

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by goanpeca** [comment]
We are in the process of creating conda packages for [napari plugins](https://www.napari-hub.org/) and related dependencies. Once this recipe passes all the checks, every time a new release goes into PyPI, a PR will be automatically open, with usually very minor need for changes so the process of maintaining this moving forward is small. Also the @conda-forge/napari team will be available to help as needed.

You can read a bit more about conda-forge at https://conda-forge.org/#contribute

--- 

@RoccoDAnt do you agree with being added as a maintainer?

**Comment by goanpeca** [comment]
Haven't heard from the author. Removing from the maintainers list. We can add them when the feedstock is created if they agree by then.

File: recipes/napari-zelda/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-zelda-{{ version }}.tar.gz
+  sha256: e4bcbcf2e844fef4acde982998f98c9f1752696becfbe838673d196da5505315
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True  # [win]
```
Comment (jaimergp) [inline recipes/napari-zelda/meta.yaml:15]:
```suggestion
  noarch: python
```

Comment (jaimergp) [reply recipes/napari-zelda/meta.yaml:15]: I think this can be noarch, let's try.

File: recipes/napari-zelda/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python
```
Comment (jaimergp) [inline recipes/napari-zelda/meta.yaml:20]:
```suggestion
    - python >=3.7
```

File: recipes/napari-zelda/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  run:
+    - datatable
+    - json5
+    - magicgui
+    - matplotlib-base >=3.4.3
+    - napari !=0.4.11
+    - napari-plugin-engine >=0.1.4
+    - numpy
+    - pandas
+    - python
```
Comment (jaimergp) [inline recipes/napari-zelda/meta.yaml:31]:
```suggestion
    - python >=3.7
```

File: recipes/napari-zelda/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    - scikit-image <=0.18.3
+    - scipy
+
+test:
+  imports:
+    - napari_zelda
```
Comment (jaimergp) [inline recipes/napari-zelda/meta.yaml:37]:
```suggestion
    - napari_zelda
    - napari_zelda.main
    - napari_zelda.napari_zelda
```

**Pull request merged**